### PR TITLE
add cloudcom2018 paper abstract

### PR DESCRIPTION
**Pull request by msimonin**

### CODE REVIEW

File: _posts/2018-09-27-scalability-and-locality-awareness-of-remote-procedure-calls.markdown [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: One paper accepted "Scalability and Locality Awareness of Remote Procedure Calls" (CloudCom 2018)
+date: 2018-09-27
+author: Javier Rojas Balderrama and  Matthieu Simonin
+categories: OpenStack CockroachDB
+---
+
+{::options parse_block_html="true" /}
```
Comment (rcherrueau) [inline _posts/2018-09-27-scalability-and-locality-awareness-of-remote-procedure-calls.markdown:9]: Could you remove this line

File: _posts/2018-09-27-scalability-and-locality-awareness-of-remote-procedure-calls.markdown [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+<div class="abstract">
```
Comment (rcherrueau) [inline _posts/2018-09-27-scalability-and-locality-awareness-of-remote-procedure-calls.markdown:10]: Idem

File: _posts/2018-09-27-scalability-and-locality-awareness-of-remote-procedure-calls.markdown [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+communication of Openstack when its agents are geo-distributed. More precisely
+we are interested in the different Remote Procedure Calls (RPCs)
+implementations of OpenStack and their behaviours with regards to three
+classical communication patterns: anycast, unicast and multicast. We discuss
+how the communication middleware can align with the geo-distribution of the RPC
+agents regarding two key factors : scalability and locality. This work
+contributes to the growing interest of adapting existing cloud computing
+infrastructures to edges models.
+
+</div>
```
Comment (rcherrueau) [inline _posts/2018-09-27-scalability-and-locality-awareness-of-remote-procedure-calls.markdown:28]: Idem

File: _posts/2018-09-27-scalability-and-locality-awareness-of-remote-procedure-calls.markdown [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+{::options parse_block_html="false" /}
```
Comment (rcherrueau) [inline _posts/2018-09-27-scalability-and-locality-awareness-of-remote-procedure-calls.markdown:29]: Idem

**Review by rcherrueau — CHANGES_REQUESTED** [review]

**Comment by rcherrueau** [comment]
It would be cool if you can add the link to the paper (thanks to a HAL ref, for instance)

**Comment by msimonin** [comment]
Closing this in favor of #6

**Comment by rcherrueau** [comment]
@msimonin @jrbalderrama please ping me when the paper will be available on HAL. I will add the link to the paper at that time. Thanks

**Comment by msimonin** [comment]
Here, 

https://hal.inria.fr/hal-01891567

----- Mail original -----

**Comment by rcherrueau** [comment]
Thanks, the post has been updated, see c149949d469328f94e7a7eb62ea1782c794287e8.

**Pull request closed without merging**